File: src/ru/gurps/generator/desktop/models/characters/CharactersAlchemy.java
package ru.gurps.generator.desktop.models.characters;

import ru.gurps.generator.desktop.config.Model;

public class CharactersAlchemy extends Model {
    public Integer id;
    public Integer characterId;
    public Integer alchemyId;

    public CharactersAlchemy() {
    }

    public CharactersAlchemy(Integer characterId, Integer alchemyId) {
        this.characterId = characterId;
        this.alchemyId = alchemyId;
    }
}

File: src/main/kotlin/ru/gurps/generator/desktop/models/characters/CharactersAlchemy.kt
package ru.gurps.generator.desktop.models.characters

import ru.gurps.generator.desktop.config.Model
import java.sql.ResultSet

class CharactersAlchemy : Model {
    override var id: Int = -1
    var characterId: Int = -1
    var itemId: Int = -1

    constructor()
    constructor(characterId: Int, itemId: Int) {
        this.characterId = characterId
        this.itemId = itemId
    }

    constructor(result: ResultSet) {
        id = result.getInt("id")
        characterId = result.getInt("characterId")
        itemId = result.getInt("itemId")
    }
}
